_data/exportendpoints.yaml

```diff
@@ -0,0 +1,9 @@
+- SFTP
```

Comment:
@judsonlmoore with this data yaml, I'm wondering if we can generate some content on a page like `ringcentral.html`

Something like:

Look at all the ways you can integrate RingCentral:
RingCentral --> SFTP
RingCentral --> Gong.io
RingCentral --> CallMiner

etc

Reply:
(again, with some automation is the question - I can just manually create this on each page if its too much effort.

Reply:
Maybe a solution could be [a chart](https://devsnap.me/css-flowcharts) with the origin on the left and then lines to each item in a stack on the right for each output possibility.
![IMG_5167](https://user-images.githubusercontent.com/52468936/172660263-7e184794-d6fd-4dfd-8652-5c87f72da2f9.jpg)